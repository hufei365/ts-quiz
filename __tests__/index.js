console.log('start test ......')

const Q = require("../dist/index").default;

const msg = "test use succes"
let q = new Q({})
q.use('parser',function(){
    return msg;
})

describe('q.run', ()=>{
    it(`it should print message: ${msg}`, ()=>{
        const msg = q.run()();
        expect(msg).toMatchSnapshot()
    })
})

console.log('end test !!!')

