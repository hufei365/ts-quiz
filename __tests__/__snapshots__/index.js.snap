// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`q.run it should print message: test use succes 1`] = `"test use succes"`;
